### PR TITLE
ET-249 Export to Common Cartridge fails for some courses

### PR DESCRIPTION
**Pull request by snowcar**

### CODE REVIEW

**Review by dmitry-viskov — APPROVED** [review]
:+1:

File: common/lib/xmodule/xmodule/modulestore/xml_exporter.py [hunk 1]
```diff
@@ -316,7 +316,8 @@ def export(self):
 
                     for s in seqs:
                         i += 1
-                        display_name = s.display_name.replace('&', '&amp;')
+                        name = s.display_name or s.display_name_with_default_escaped
+                        display_name = name.replace('&', '&amp;')
```
Comment (dpowell) [inline common/lib/xmodule/xmodule/modulestore/xml_exporter.py:320]: Do we need a more comprehensive HTML encoding? (For things like `>`, etc?)

Comment (olaputin) [reply common/lib/xmodule/xmodule/modulestore/xml_exporter.py]: I think, yes. I'll investigate this and test. Thanks for your point.

Comment (snowcar) [reply common/lib/xmodule/xmodule/modulestore/xml_exporter.py:320]: I think, yes. I'll investigate this and test. Thanks for your point.

**Pull request merged**